annotator_store/views.py

```diff
@@ -236,6 +236,17 @@ def get(self, request):
                 )
                 # NOTE: contains search on extra data jsonfield is
                 # probably not a great idea...
+            # non-standard field search on extra data
```

Comment:
Need to add documentation to the view method about this type of search being supported